pkgs/development/r-modules/default.nix

```diff
@@ -1318,6 +1318,43 @@ let
       buildInputs = [ cacert ] ++ attrs.buildInputs;
     });
 
+
+    immunotation = let
+      MHC41alleleList = fetchurl {
+        url = "https://services.healthtech.dtu.dk/services/NetMHCpan-4.1/allele.list";
+        hash = "sha256-CRZ+0uHzcq5zK5eONucAChXIXO8tnq5sSEAS80Z7jhg=";
+      };
+
+      MHCII40alleleList = fetchurl {
+        url = "https://services.healthtech.dtu.dk/services/NetMHCIIpan-4.0/alleles_name.list";
+        hash = "sha256-K4Ic2NUs3P4IkvOODwZ0c4Yh8caex5Ih0uO5jXRHp40=";
+      };
```

Comment:
Are these stable or should they be archived on the web archive?

Reply:
The database is not updated very frequently, last update was in 2020. But even when they do update it, you can still access the previous versions. For example, https://services.healthtech.dtu.dk/services/NetMHCIIpan-3.2/alleles_name.list, from 2018. So as long as the server is alive, I don't see a need to point to web archive.